assets/src/hooks/useDetour.tsx

```diff
@@ -67,11 +67,18 @@ export const useDetour = () => {
   }
 
   const canUndo =
-    startPoint !== null && endPoint === null && waypoints.length > 0
+    startPoint !== null || endPoint !== null || waypoints.length > 0
```

Comment:
**nitpick:** since we now allow undoing in all states after the detour has been started, I think you can simplify this to:

```suggestion
    startPoint !== null
```

Reply:
[Touché](https://github.com/mbta/skate/pull/2417/commits/a25229a6034193d6cc76bf485099fb65871c4973)